reconcile/utils/jjb_client.py

```diff
@@ -185,7 +185,9 @@ def print_diff(self, files, replace_path, action):
                 if equal:
                     continue
 
-            instance, item, _ = f.replace(replace_path + "/", "").split("/")
+            instance, *items, _ = f.replace(replace_path + "/", "").split("/")
+            item = "_".join(items)
```

Comment:
Is there a specific reason to replace the original app-interface defined value with `_`? My understanding is `item` is only used for logging right? 

```
instance, item, _ = f.replace(replace_path + "/", "").split("/", 2)
```

That way you could still log the original branch name defined in app-interface.

Reply:
It actually creates a directory structure names after the branch, so it will create a nested directory in the case above without replacing the value:
```
ls
testbuild-with-slash-FINDME/BRANCH/config.xml
```

Reply:
jjb_client.py uses a wrapper class around the [jenkins-job-builder project](https://jenkins-job-builder.readthedocs.io/en/latest/index.html), so `item` is used in `builder.py` from this project, and inside the `JenkinsManager` class that is imported: `from jenkins_jobs.builder import JenkinsManager`.

Debug output from the same repo configuration containing a `/`:
```
_setup_output: testbuild-with-slash-FINDME/BRANCH
```